chromium_src/third_party/blink/renderer/modules/speech/speech_synthesis.cc

```diff
@@ -0,0 +1,58 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * you can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "third_party/blink/renderer/modules/speech/speech_synthesis.h"
+#include "brave/third_party/blink/renderer/brave_farbling_constants.h"
+#include "third_party/blink/public/platform/web_content_settings_client.h"
+
+#define OnSetVoiceList OnSetVoiceList_ChromiumImpl
+#include "../../../../../../../third_party/blink/renderer/modules/speech/speech_synthesis.cc"
+#undef OnSetVoiceList
+
+namespace blink {
+
+void SpeechSynthesis::OnSetVoiceList(
+    Vector<mojom::blink::SpeechSynthesisVoicePtr> mojom_voices) {
+  voice_list_.clear();
+  BraveFarblingLevel farbling_level = BraveFarblingLevel::OFF;
+  if (ExecutionContext* context = GetExecutionContext()) {
```

Comment:
nit: seems like we should have some kind of helper to get the farbling level

Reply:
Logged issue in https://github.com/brave/brave-browser/issues/19318